languages/julia/exercises/concept/functions-introduction/.docs/introduction.md

```diff
@@ -24,15 +24,13 @@ The latter is most commonly used for one-line function definitions or mathematic
 Invoking a function is done by specifying its name and passing arguments for each of the function's parameters:
 
 ```julia
-add(x, y) = x + y
-mul(x, y) = x * y
-muladd(x, y, z) = add(mul(x, y), z)
+muladd(10, 5, 1)
```

Comment:
I used the "functions invoking functions" example because it's closer to the expected solution for this exercise. What do you think about having both?

Reply:
Yes, we could have both. I thought it was a bit confusing for the reader that the prose says "this is how you invoke functions", and the code has three function definitions in it.

Reply:
```suggestion
add(x, y) = x + y
mul(x, y) = x * y
muladd(x, y, z) = add(mul(x, y), z)
muladd(10, 5, 1)
```

Reply:
How about something like this?

```julia
# invoking a function
muladd(10, 5, 1)

# and of course you can invoke a function within the body of another function:
square_plus_one(x) = muladd(x, x, 1)
```

Reply:
sounds good

Reply:
```suggestion
# invoking a function
muladd(10, 5, 1)

# and of course you can invoke a function within the body of another function:
square_plus_one(x) = muladd(x, x, 1)
```